styles/components/Icon.scss

```diff
@@ -1,6 +1,22 @@
 .Icon {
-  width: 1em;
-  height: 1em;
+  width: 2em;
```

Comment:
why change default size to 2em?

Reply:
with `{width,height}:1em` you can just change font-size to 2em,0.5em and size will be grown based on font-size